syntax = "proto3";

service Elector
{
    rpc elect_commander(request) returns (elected_commander) {}

}

message request
{
    int64 dummy=1;
}
message elected_commander
{
    int64 soldier_number = 1;
    int64 x_pos = 2;
    int64 y_pos = 3;
    int64 speed_capacity = 4;
}
